webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -0,0 +1,42 @@
+import Jed from 'jed';
+import { deprecateObjectProperty } from '../../foreman_tools';
+
+class I18n {
```

Comment:
I wonder if we actually need a class constructor? (since there is only one instance?) I'm OK with keeping it if it makes our life easier.